ci-operator/step-registry/terraform/aws/OWNERS

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/terraform-redhat/terraform-provider-ocm root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
delete. this OWNERS file is not really auto-generated
same for the other places